index.html

```diff
@@ -5,15 +5,198 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Coda+Caption:800|Fira+Sans&display=swap" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" type="text/css" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
+    <header>
+        <div class="container">
+
+            <nav class="nav-bar row">
+
+                <div class="col-3 logo-div">
+                    <img src="img/james smith.png" alt="logo" class="logo">
+                </div>
+
+                <ul class="col-7 menu">
+                    <li>
+                        <a href="#">Home</a>
+                    </li>
+
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+
+                    <li>
+                        <a href="#">Services</a>
+                    </li>
+
+                    <li>
+                        <a href="#">Motivational Reading</a>
+                    </li>
+
+                    <li>
+                        <a href="#">Blog</a>
+                    </li>
+
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                </ul>
+
+                <div class="col-2 login">
+                    <i class="fa fa-user-circle"></i>
+                    <a href="#">Log In</a>
+                </div>
+
+            </nav>
+
+        </div>
+        </div>
```

Comment:
Be careful with extra closing tags that have not been opened (or vice-versa). Important to keep indentation always correct to make it easier to spot them